roles/identity-management/manage-rh-sso-identities/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+rh_sso_host: "192.168.16.3"
```

Comment:
Not sure that `rh_sso_host` belongs under defaults (as this is a bit of a required thing that will change from env to env). Would like to see this set in the example inventory and then defined better (as well as the rest of our options) in a `README`.

Reply:
@oybed Interested in your thoughts here.

Reply:
Agreed on having a "fixed" IP in the role doesn't make sense. This should be one of those variables that are documented in the README as *required*.